### DISCUSSION TITLE
Change color of snap area footprint

### DISCUSSION THREAD

**Discussion by mfvd** [rxhanson/Rectangle#373] [Q&A] [answered]
Would it be possible to change the color, not just the transparency, of the snap area footprint?

I'm on a black background and under certain illumination conditions it's really difficult to see where/how the snapping will go (footprint).

**Comment by rxhanson** [comment]
This has been on my Todo list for a little while, I just haven't had time to add it in. I will probably add this in as a terminal command config at first.

**Comment by mfvd** [comment]
Nice, I was actually asking if something along terminal config that could be used.
By the way... really cool app. Thanks!

**Comment by rxhanson** [comment, marked as answer]
Alright, this is now in v0.43.

The terminal command is in this section of the readme:
https://github.com/rxhanson/Rectangle#modify-the-footprint-displayed-for-drag-to-snap-area

**Comment by benheymink** [comment]
For those looking, the options are documented here now: https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#modify-the-footprint-displayed-for-drag-to-snap-area
